compiler/src/compiler/ast_to_hir.rs

```diff
@@ -309,6 +310,28 @@ impl<'a> Context<'a> {
         }
     }
 
+    fn lower_text(&mut self, id: Option<ast::Id>, text: &Text) -> hir::Id {
+        // TODO: Convert parts to text
+        let concatenate = self.push(
+            None,
+            Expression::Builtin(BuiltinFunction::TextConcatenate),
+            None,
+        );
+        let mut result = self.push(id, Expression::Text("".to_string()), None);
```

Comment:
We can get rid of this expression for all non-empty texts by using reduce, similar to here: https://github.com/candy-lang/candy/blob/220712fdd89458ff58e754a102526cf133830601/compiler/src/compiler/hir_to_mir.rs#L703-L715

(But using `unwrap_or_else(…)` instead of `unwrap()`, since the parts may be empty)

Reply:
Resolved by d87609c